action.yml

```diff
@@ -21,6 +21,9 @@ inputs:
     description: 'Path to where the compressed JDK is located. The path could
        be in your source repository or a local path on the agent.'
     required: false
+  distribution:
+    description: 'Vendor providing of java distribution'
```

Comment:
```suggestion
    description: 'Java distributor'
```

Reply:
Maybe also makes sense to reference the list of supported distributors, like `description: 'Java distributor. See the list of supported distributors in README file.'`

Reply:
Done. Thank you.